src/.golangci.yml

```
@@ -34,6 +34,9 @@ linters-settings:
       - '\bpath\.(Ext|Base|Dir|Join)'
       # Don't allow the typo m356 to be used in place of m365.
       - '[Mm]356'
+      # Prefer suite.Run(name, func() {}) for subtests as testify has it instead
+      # of suite.T().Run(name, func(t *testing.T) {}).
+      - 'T\(\)\.Run(# prefer testify subtest Run(name, func()) function )?'
```

Might want the following, as the current regex doesn't catch t.Run().
```suggestion
      - '(T\(\)|t)\.Run(# prefer testify suite.Run(name, func()) )?'
```